src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -110,6 +112,9 @@
  *
  *      synchronized(this) { } // violation ''synchronized' is not followed by whitespace'
  *      synchronized (this) { } // ok
+ *
+ *      return (x * x); // OK
+ *      return(x * x); // violation ''return' is not followed by whitespace'
```

Comment:
Please do not use non-compilable code in examples. You can use a couple of simple lambdas here to show check behavior for return statements.

Reply:
Changes Done

Reply:
Not done. A void method in Java can not have a return statement that returns a value, and no method in Java can have a method with two returns. The second return statement is always unreachable, and will not compile.

Reply:
Please check now.

Reply:
Still not done. See https://github.com/checkstyle/checkstyle/pull/11538#discussion_r847958594 .

Reply:
@Rahulkhinchi03 I see that you changed example. Please make it with parens as before. According to report, it is most common case.

Reply:
@strkkk I am using `return(x * x)` as an example. Do you want me to use an example with Lambda? If yes, I was getting error of bad use.

Reply:
You do not have to use a lambda, but having unreachable/ non-compilable code in an example is not good. 

You could also add two very simple methods to this example, one ok and one with violation.

Reply:
@Rahulkhinchi03 please change 2 consecutive returns to
```
String m1() {
   return ("a" + "b");
}

String m2() {
   return("a" + "b");
}
```

Reply:
Okay I am doing the changes but please guide me: What is the problem with my example?

Reply:
@Rahulkhinchi03 if you ever have doubts, create a class with code example like this and try to compile it. javac will tell you where your errors are.

Reply:
Changes Done

Reply:
> create a class with code example like this and try to compile it

Please do this.

Reply:
@nick-mancuso I have done that. It says that the result is ignored. Thank you for making me understand!

Reply:
@Rahulkhinchi03 please do https://github.com/checkstyle/checkstyle/pull/11538#discussion_r849756475

Reply:
Please check. It is already done.

Reply:
Make sure that you have pushed your changes, non compilable code is still in documentation.

Reply:
I had a different example in the documentation before then I changed my code to https://github.com/checkstyle/checkstyle/pull/11538#discussion_r849756475 after the review.

Reply:
I can see that you have changed the documentation, but you did not use the examples given to you at https://github.com/checkstyle/checkstyle/pull/11538#discussion_r849756475 and comments at https://github.com/checkstyle/checkstyle/pull/11538#discussion_r847958594 still apply.

Reply:
I am using the return in void method but not returning a value. For documentation, you mean to  say that instead of just using   
```
return("a" + "b");  // violation ''return' is not followed by whitespace'
return ("a" + "b");  // ok
```

I should use this in documentation:
```
    String testOne() { return ("a" + "b");  } // ok
    String testOne() { return("a" + "b");  } // violation ''return' is not followed by whitespace'
```

Reply:
Resolved.